IDE/OPENSTM32/Inc/user_settings.h

```diff
@@ -22,8 +22,8 @@ extern "C" {
 #undef  WOLFSSL_STM32F4
 #define WOLFSSL_STM32F4
 
-#undef  WOLFSSL_STM32_CUBEMX
-#define WOLFSSL_STM32_CUBEMX
+#undef  WOLFTPM_STM32_CUBEMX_SPI
```

Comment:
You need to keep `WOLFSSL_STM32_CUBEMX`. That is a define we already use in wolfSSL to indicate using STM32 with the CubeMX HAL. We already have defines in wolfTPM to indicate I2C using `WOLFTPM_I2C`. Please revert this and default to SPI unless `WOLFTPM_I2C` is defined for Cube.

Reply:
Add the code flow will be:

```
#ifdef WOLFSSL_STM32_CUBEMX
#ifdef WOLFTPM_I2C
/* Use STM32CubeHAL with I2C */
#else
/* Use STM32CubeHAL with SPI */
#endif /* WOLFTPM_I2C */
#endif /* WOLFSSL_STM32_CUBEMX */
```

Reply:
Thansk, David. It is now fixed.

I did not know WOLFSSL_STM32_CUBEMX is used across wolfSSL and wolfTPM.

I also got lost in the long file and thought WOLFTPM_I2C is nested with the `__linux__` ifdefs.